terraform-plans/templates/github/SECURITY.md.tftpl

```diff
@@ -0,0 +1,24 @@
+<!-- This file is centrally managed as a template file in https://github.com/canonical/solutions-engineering-automation -->
+<!-- To update the file: -->
+<!-- - Edit it in the canonical/solutions-engineering-automation repository. -->
+<!-- - Open a PR with the changes. -->
+<!-- - When the PR merges, the soleng-terraform bot will open a PR to the target repositories with the changes. -->
+
+# Security policy
+
+If the vulnerability affects a dependency, a new version of the component including the updated
```

Comment:
Maybe `project` instead of the component, it sounds like you referring to the dependency

Reply:
If we reword it here, does it also need to be updated in our spec? Perhaps changes need to go through the spec first?

Reply:
The spec is already approved by the security team, so I don't see the need of it

Reply:
Agreed, please revert the re-word; we can discuss changes to the wording on the spec later. :)